test/terraform/permanent/providers.tf

```diff
@@ -0,0 +1,30 @@
+terraform {
+  required_version = "1.9.8"
+  required_providers {
+    aws = {
+      version = "5.76.0"
+    }
+  }
+}
+
+terraform {
+  backend "s3" {
+    # no variables allowed - so account_id and region hardcoded
+    encrypt        = true
+    bucket         = "nr-terraform-states-571600857950"
+    dynamodb_table = "terraform-states-lock"
+    region         = "us-east-1"
+    key            = "newrelic/opentelemetry-collector-releases/permanent/terraform.tfstate"
+    role_arn       = "arn:aws:iam::571600857950:role/resource-provisioner"
```

Comment:
I don't think we want to expose the account number since the repo is public, thoughts on using a partial config? https://developer.hashicorp.com/terraform/language/backend#partial-configuration

Reply:
Had forgotten about partial configs. Totally agree!